Templates/Ros2ProjectTemplate/Template/Gem/Source/${Name}SampleComponent.h

```diff
@@ -0,0 +1,40 @@
+
```

Comment:
Missing copyright header
```suggestion

// {BEGIN_LICENSE}
/*
 * Copyright (c) Contributors to the Open 3D Engine Project.
 * For complete copyright and license terms please see the LICENSE at the root of this distribution.
 *
 * SPDX-License-Identifier: Apache-2.0 OR MIT
 *
 */
// {END_LICENSE}
```